packages/hooks/src/cache.ts

```diff
@@ -0,0 +1,45 @@
+import { LRUCache } from 'lru-cache';
+
+import { EncreData } from './types';
+
+export interface EncreCache {
```

Comment:
What is this interface for specifically, I understand that it is storing a graph and a node, but just want to make sure I understand the software, what part fo the process does EncreCache speeds up specifically?

Reply:
Good Question! `EncreCache` is used for storing incoming data from input handlers. You can image a scenario where there is a message box that connects to an input handler, so whenever user change the message, the input handler should store that changed value. But this changed value is not sent to the workflow immediately, since no action is triggered by the uset. So we need a cache to store this value.